lib/utilities/strategy-helpers/smi-canary-deployment-helper.js

```diff
@@ -145,24 +147,27 @@ function adjustTraffic(kubectl, manifestFilePaths, stableWeight, canaryWeight) {
     core.debug('serviceObjects:' + serviceObjects.join(',') + ' result:' + result);
     utility_1.checkForErrors([result]);
 }
-function updateTrafficSplitObject(serviceName) {
+function updateTrafficSplitObject(kubectl, serviceName) {
     const percentage = parseInt(TaskInputParameters.canaryPercentage) * 10;
     const baselineAndCanaryWeight = percentage / 2;
     const stableDeploymentWeight = 1000 - percentage;
     core.debug('Creating the traffic object with canary weight: ' + baselineAndCanaryWeight + ',baseling weight: ' + baselineAndCanaryWeight + ',stable: ' + stableDeploymentWeight);
-    return createTrafficSplitManifestFile(serviceName, stableDeploymentWeight, baselineAndCanaryWeight, baselineAndCanaryWeight);
+    return createTrafficSplitManifestFile(kubectl, serviceName, stableDeploymentWeight, baselineAndCanaryWeight, baselineAndCanaryWeight);
 }
-function createTrafficSplitManifestFile(serviceName, stableWeight, baselineWeight, canaryWeight) {
-    const smiObjectString = getTrafficSplitObject(serviceName, stableWeight, baselineWeight, canaryWeight);
+function createTrafficSplitManifestFile(kubectl, serviceName, stableWeight, baselineWeight, canaryWeight) {
+    const smiObjectString = getTrafficSplitObject(kubectl, serviceName, stableWeight, baselineWeight, canaryWeight);
     const manifestFile = fileHelper.writeManifestToFile(smiObjectString, TRAFFIC_SPLIT_OBJECT, serviceName);
     if (!manifestFile) {
         throw new Error('UnableToCreateTrafficSplitManifestFile');
     }
     return manifestFile;
 }
-function getTrafficSplitObject(name, stableWeight, baselineWeight, canaryWeight) {
+function getTrafficSplitObject(kubectl, name, stableWeight, baselineWeight, canaryWeight) {

```

Comment:
Add an extra line between functions

Reply:
js files are automatically generated during build and there's no extra line between any of the functions in any file. Should I still do it for this function?

Reply:
No no. I mistook it for a ts file; you can ignore this comment